PlatformConfig.mk

```diff
@@ -43,19 +43,13 @@ BOARD_KERNEL_CMDLINE += androidboot.bootdevice=7824900.sdhci
 
 TARGET_RECOVERY_FSTAB ?= $(PLATFORM_COMMON_PATH)/rootdir/vendor/etc/fstab.loire
 
-# Wi-Fi definitions for Broadcom solution
-BOARD_WLAN_DEVICE           := bcmdhd
+# Wi-Fi definitions for Broadcom solution but using brcmfmac instead of bcmdhd kernel driver
```

Comment:
I would explain here why qcwcn is used if using brcmfmac  as kernel driver, which seems counterintuitive and it's explained in the commit message.